### ISSUE TITLE
macOS AppleClang compilation error "no member named 'srgb'"

### ISSUE THREAD

**Issue by agriggio** [artpixls/ART#305]
**[Original report](https://bitbucket.org/agriggio/art/issues/305) by Richard Barber (Bitbucket: [Richard Barber](https://bitbucket.org/Richard Barber), ).**

----------------------------------------

```
/Users/rb/art/rtengine/improcfun.cc:106:75: error: no member named 'srgb' in 'rtengine::Settings'

        monitor = ICCStore::getInstance()->getProfile (options.rtSettings.srgb);

                                                       ~~~~~~~~~~~~~~~~~~ ^
```

I’m stuck on this error at 17% through.

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

fixed compilation error on OSX

Fixes #305

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

sorry, my bad. Should be fixed now, thanks!
